config/runtime.exs

```diff
@@ -5,6 +5,16 @@ import Config
 # config :livebook, :node, {:shortnames, "livebook"}
 # config :livebook, :node, {:longnames, :"livebook@127.0.0.1"}
 
+# Note that presence of `USE_PASSWORD` will overwrite already set `:token_authentication`
+if System.get_env("USE_PASSWORD") do
+  config :livebook,
+    token_authentication: false,
+    password_authentication: true,
+    password: System.fetch_env!("AUTH_PASSWORD")
+else
+  config :livebook, password_authentication: false
+end
```

Comment:
I like the suggestion of having a single `:authentication_mode` configuration that is either token, password, or disabled. Other than that,  I think a single env var would suffice:

```suggestion
if password = System.get_env("LIVEBOOK_PASSWORD") do
  config :livebook,
    authentication_mode: :password,
    password: password
else
  config :livebook, :authentication_mode, :token
end
```